lib/components/table.vue

```diff
@@ -524,14 +533,25 @@
 
                 if (data.then && typeof data.then === 'function') {
                     // Provider returned Promise
-                    data.then((items) => {
+                    data.then(items => {
                         this._providerSetLocal(items);
                     });
                 } else {
                     // Provider returned Array data
                     this._providerSetLocal(data);
                 }
+            },
+            hasFormatter(field) {
+                return field.formatter && ((typeof (field.formatter) === 'function') || (typeof (field.formatter) === 'string'));
+            },
+            callFormatter(item, key, field) {
+                if (field.formatter && (typeof (field.formatter) === 'function'))
+                    return field.formatter(item[key]);
+
+                if (field.formatter && (typeof (this.$parent[field.formatter]) === 'function'))
+                    return this.$parent[field.formatter](item[key]);
```

Comment:
So, just making a guess... does this allow calling a method defined in the parent (when field.formatter is a string)?